tools/logicalimager/Version.h

```diff
@@ -0,0 +1,68 @@
+/*
+** The Sleuth Kit
+**
+** Brian Carrier [carrier <at> sleuthkit [dot] org]
+** Copyright (c) 2010-2019 Brian Carrier.  All Rights reserved
+**
+** This software is distributed under the Common Public License 1.0
+**
+*/
+
+/**
+* \file Version.h
+* Contains the class definitions for the Version class
+*/
+
+#pragma once
+
+#include <string>
+
+/**
+* Implement the major and minor version.
+*
+*/
+class Version {
+public:
+
+    Version(std::string version)
+    {
+        int status = std::sscanf(version.c_str(), "%d.%d", &m_major, &m_minor);
+        if (status != 2) {
+            throw std::logic_error("ERROR: Invalid version " + version + ". Expected major.minor");
+        }
+    }
+
+    bool isSupported(const Version& other) const {
+        return m_major >= other.m_major;
+    }
+
+    std::string getSupportedVersion() const {
+        std::string str;
+        str = std::to_string(m_major) + ".x";
+        return str;
+    }
+
+    bool operator < (const Version& other) const {
+        if (m_major < other.m_major)
+            return true;
+        if (m_minor < other.m_minor)
+            return true;
+        return false;
```

Comment:
This logic doesn't look correct. If we're comparing 3.1 with 2.2 (other), it looks like it will return true.